pyproject.toml

```diff
@@ -23,11 +22,10 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
 requests = "^2.23"
 cachetools = "^5.3"
 rsa = "^4.0"
-importlib-metadata = {version = ">=3.6", python = "<3.8"}
```

Comment:
can you remove this `except` block also?

https://github.com/mdsol/mauth-client-python/blob/cc956ffb2c11094f6ceb535c86716d61cfe25a1f/mauth_client/__init__.py#L4-L6

Reply:
thank you!!